src/main/controllers/DailyCauseListController.ts

```diff
@@ -43,6 +43,7 @@ export default class DailyCauseListController {
             res.render(listToLoad, {
                 ...cloneDeep(req.i18n.getDataByLanguage(pageLanguage)[listToLoad]),
```

Comment:
I think we're missing language file updates for the Civil, Family and Mixed lists to remove the open justice override

Reply:
Added the missing language resource files with old open justice statement removed